DataFormats/Common/interface/DetSetVectorNew.h

```diff
@@ -427,9 +440,19 @@ namespace edmNew {
       // delete content if T is pointer...
     }
 
+    // Implement copy constructor because of a (possibly temporary)
+    // need in heterogeneous framework prototyping. In general this
+    // class is still supposed to be non-copyable, so to prevent
+    // accidental copying the assignment operator is left deleted.
+    DetSetVector(const DetSetVector& rh):
+      DetSetVectorTrans(rh),
+      m_subdetId(rh.m_subdetId),
+      m_ids(rh.m_ids),
+      m_data(rh.m_data)
+    {}
+
     // default or delete is the same...
     DetSetVector& operator=(const DetSetVector&) = delete;
-    DetSetVector(const DetSetVector&) = delete;
```

Comment:
Would allowing the default instead of hand making the constructor be enough? You could still have the comment.

Reply:
No, because the base class (`DetSetVectorTrans`) has two `std::atomic` members. There are comments for `DetSetVectorTrans`, I can copy them here as well if you want.

Reply:
Spoke too soon, of course the copy constructor here can be defaulted (as are the moves).